packages/pn-personafisica-webapp/src/App.tsx

```diff
@@ -236,8 +236,12 @@ const ActualApp = () => {
   });
 
   const handleUserLogout = () => {
+    const isPpOrTosPage =
```

Comment:
@alessandrogelmi let's try to generalize this behaviour for all public pages, we can make al list or a map containing public routes related to the single webapp and an utility method (e.g. isPublicPage) that returns true o false

Reply:
After an analysis, I discovered that removing the forced login redirect (goToLoginPortal function) resolves everything. Logging out from pages that are not within the SessionGuard automatically stays on that page.